<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateCalculateDialog</class>
 <widget class="QDialog" name="CreateCalculateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="0">
    <widget class="QPushButton" name="asinPushButton">
     <property name="text">
      <string>asin</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="kHatPushButton">
     <property name="text">
      <string>kHat</string>
     </property>
    </widget>
   </item>
   <item row="7" column="5">
    <widget class="QPushButton" name="floorPushButton">
     <property name="text">
      <string>floor</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="iHatPushButton">
     <property name="text">
      <string>iHat</string>
     </property>
    </widget>
   </item>
   <item row="9" column="4">
    <widget class="QPushButton" name="expPushButton">
     <property name="text">
      <string>exp</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="dividedPushButton">
     <property name="text">
      <string>/</string>
     </property>
    </widget>
   </item>
   <item row="11" column="3">
    <widget class="QPushButton" name="normPushButton">
     <property name="text">
      <string>norm</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="minusPushButton">
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="9" column="3">
    <widget class="QPushButton" name="superscriptPushButton">
     <property name="text">
      <string>x^y</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="cosPushButton">
     <property name="text">
      <string>cos</string>
     </property>
    </widget>
   </item>
   <item row="11" column="4">
    <widget class="QPushButton" name="backPushButton">
     <property name="text">
      <string>back</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0" colspan="6">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QPushButton" name="minPushButton">
     <property name="text">
      <string>min</string>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QPushButton" name="logPushButton">
     <property name="text">
      <string>log10</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QPushButton" name="atanPushButton">
     <property name="text">
      <string>atan</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QPushButton" name="acosPushButton">
     <property name="text">
      <string>acos</string>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QPushButton" name="tanhPushButton">
     <property name="text">
      <string>tanh</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QPushButton" name="sinhPushButton">
     <property name="text">
      <string>sinh</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="multiplyPushButton">
     <property name="text">
      <string>*</string>
     </property>
    </widget>
   </item>
   <item row="7" column="4">
    <widget class="QPushButton" name="maxPushButton">
     <property name="text">
      <string>max</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QPushButton" name="lnPushButton">
     <property name="text">
      <string>ln</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="tanPushButton">
     <property name="text">
      <string>tan</string>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <widget class="QPushButton" name="closeParenPushButton">
     <property name="text">
      <string>)</string>
     </property>
    </widget>
   </item>
   <item row="11" column="5">
    <widget class="QPushButton" name="clearPushButton">
     <property name="text">
      <string>clear</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QPushButton" name="sqrtPushButton">
     <property name="text">
      <string>sqrt</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="absPushButton">
     <property name="text">
      <string>abs</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="6">
    <widget class="QLineEdit" name="expressionLineEdit"/>
   </item>
   <item row="2" column="3" colspan="3">
    <widget class="QPushButton" name="vectorPushButton">
     <property name="text">
      <string>Vectors</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="sinPushButton">
     <property name="text">
      <string>sin</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="addPushButton">
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QPushButton" name="pointPushButton">
     <property name="text">
      <string>v1.v2</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QPushButton" name="openParenPushButton">
     <property name="text">
      <string>(</string>
     </property>
    </widget>
   </item>
   <item row="5" column="5">
    <widget class="QPushButton" name="ceilPushButton">
     <property name="text">
      <string>ceil</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="jHatPushButton">
     <property name="text">
      <string>jHat</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QPushButton" name="crossPushButton">
     <property name="text">
      <string>cross</string>
     </property>
    </widget>
   </item>
   <item row="11" column="2">
    <widget class="QPushButton" name="magPushButton">
     <property name="text">
      <string>mag</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QPushButton" name="coshPushButton">
     <property name="text">
      <string>cosh</string>
     </property>
    </widget>
   </item>
   <item row="9" column="5">
    <widget class="QPushButton" name="dotPushButton">
     <property name="text">
      <string>dot</string>
     </property>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="QPushButton" name="signPushButton">
     <property name="text">
      <string>sign</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QPushButton" name="scalarPushButton">
     <property name="text">
      <string>Scalars</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="nameLineEdit"/>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4" colspan="2">
    <widget class="QComboBox" name="typeComboBox">
     <item>
      <property name="text">
       <string>Point Data</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Cell Data</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CreateCalculateDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CreateCalculateDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
